site/src/docs/patterns/navigation/building-navigation.mdx

```diff
@@ -0,0 +1,522 @@
+    The learn more about the <strong>Header</strong> component, please refer to
+    <InternalLink href="/components/header">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 2. Breadcrumbs
+
+Breadcrumbs are a secondary navigation component that helps users understand their current location within the site hierarchy and offers a simple way to navigate back to previous pages. Placed immediately below the header, breadcrumbs provide context and enhance usability by offering a clear path through the website or application.
+
+<PlaygroundPreview>
+  <Breadcrumb
+    ariaLabel="Breadcrumb"
+    list={[
+      { title: 'Front page', path: '/' },
+      { title: 'Health and social services', path: '/path' },
+      { title: 'Senior services', path: '/path/2ndLevelPath' },
+      { title: 'Informal care', path: '/path/2ndLevelPath/3rdLevelPath' },
+      { title: 'Care options', path: null },
+    ]}
+  ></Breadcrumb>
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>Breadcrumb</strong> component, please refer to
+    <InternalLink href="/components/breadcrumb">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 3. Hero
+
+The hero section is an optional but highly recommended component that usually appears below the breadcrumbs. It serves as a visual and content-focused introduction to the page, typically including a large image, heading, and a call-to-action button or message. The hero section helps grab the user's attention and sets the tone for the rest of the page.
+
+<PlaygroundPreview>
+  <Hero
+    imageSrc="https://hds.hel.fi/images/homepage/amos58.jpg"
+    variant="diagonalKoros"
+    style={{
+      '--background-color': 'var(--color-coat-of-arms)',
+      '--color': '#fff',
+      '--koros-color': 'var(--background-color)',
+      '--diagonal-koros-position': '55%',
+      '--koros-height': '14px',
+    }}
+  >
+    <Hero.Title>Hero title</Hero.Title>
+    <Hero.Text>
+      This is an introduction text. The length is circa two sentences. The suitable total length of the title and
+      introduction is around 230 characters, including spaces.
+    </Hero.Text>
+    <Button
+      variant="primary"
+      role="link"
+      style={{ '--background-color': '#fff', '--color': '#000', '--border-color': '#fff' }}
+    >
+      Call to action
+    </Button>
+  </Hero>
+</PlaygroundPreview>
+
+<ul style="margin-bottom:0;">
+  <li>
+    The learn more about the <strong>Hero</strong> component, please refer to
+    <InternalLink href="/components/hero">the component page.</InternalLink>
+  </li>
+</ul>
+
+### 4. Side navigation (optional)
+
+Side navigation can serve as a secondary navigation tool alongside the main navigation components at the top of a page. Side navigation is useful when the application or website has a complex or multi-layered hierarchy of information, where the top-level navigation isn't sufficient to provide a complete, comprehensive navigation experience.
```

Comment:
Side navigation should not be used on landing pages with hero.

Reply:
Yes, that's a guideline that should be mentioned here.